Mathlib/MeasureTheory/Integral/IntervalIntegral.lean

```diff
@@ -101,6 +101,14 @@ theorem intervalIntegrable_iff_integrable_Icc_of_le {f : ℝ → E} {a b : ℝ}
   rw [intervalIntegrable_iff_integrable_Ioc_of_le hab, integrableOn_Icc_iff_integrableOn_Ioc]
 #align interval_integrable_iff_integrable_Icc_of_le intervalIntegrable_iff_integrable_Icc_of_le
 
+theorem intervalIntegrable_iff_integrable_Ico_of_le [NoAtoms μ] (hab : a ≤ b) :
+    IntervalIntegrable f μ a b ↔ IntegrableOn f (Ico a b) μ := by
+  rw [intervalIntegrable_iff_integrable_Icc_of_le hab, integrableOn_Icc_iff_integrableOn_Ico]
+
+theorem intervalIntegrable_iff_integrable_Ioo_of_le [NoAtoms μ] (hab : a ≤ b) :
+    IntervalIntegrable f μ a b ↔ IntegrableOn f (Ioo a b) μ := by
```

Comment:
I think this might be a better statement if you reverse it? It could even be simp, although I doubt this would fire often.

Reply:
There are already several lemmas of this form, and in the library they are used roughly 50% of the time from left to right, 50% of the time from right to left, so I don't think there is a good reason to change here.